compiler/frontend/beam/pom.xml

```diff
@@ -27,6 +27,7 @@ under the License.
 
   <artifactId>nemo-compiler-frontend-beam</artifactId>
   <name>Nemo Compiler Frontend: Beam</name>
+  <packaging>jar</packaging>
```

Comment:
What's this for?

Reply:
This has been how it's been before. For running the `run_beam.sh` script on YARN environments, I experienced errors without having them as über-jars.

Reply:
Could you please revert this change? 
`run_beam.sh` includes client.jar and we don't need nemo-example jar file. You should include the client jar file, instead of beam jar.  Also, nemo-360 is not related to this change. Please create another PR for separate issue.

Reply:
Done.